### PR TITLE
fix load image from i0.wp.com komikcast

### PR DESCRIPTION
**Pull request by arifr007**
add custom header when load image from i0.wp.com

### CODE REVIEW

**Comment by rama982** [comment]
Can you revert that unnecessary changes on filters code?  for clean diff

**Comment by arifr007** [comment]
@rama982 done

File: src/id/komikcast/src/eu/kanade/tachiyomi/extension/id/komikcast/Komikcast.kt [hunk 1]
```diff
@@ -197,11 +197,26 @@ class Komikcast : ParsedHttpSource() {
...
+        var headers = Headers.Builder()
+        headers.apply {
             add("Referer", baseUrl)
-        }.build()
-        return GET(page.imageUrl!!, imgHeader)
+            add("User-Agent", "Mozilla/5.0 (Linux; U; Android 4.4.2; en-us; LGMS323 Build/KOT49I.MS32310c) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/76.0.3809.100 Mobile Safari/537.36")
+        }
+
+        if (page.imageUrl!!.contains("i0.wp.com")) {
+            headers.apply {
```
Comment (arkon) [inline src/id/komikcast/src/eu/kanade/tachiyomi/extension/id/komikcast/Komikcast.kt]: Are all of these actually necessary?

Comment (arifr007) [reply src/id/komikcast/src/eu/kanade/tachiyomi/extension/id/komikcast/Komikcast.kt]: without this header image from i0.wp.com wont show idk why

**Comment by rama982** [comment]
After some test only "Accept" header resulting the image, other is unnecessary.

**Comment by rama982** [comment]
After testing only "Accept" header resulting the image.

Test with :

```
curl -X GET -H 'Accept: image/webp' -v -i 'http://i0.wp.com/lh3.googleusercontent.com/-gUEesyY5dCc/Wvwh-6lI3PI/AAAAAAAApPg/sbHUwBWmB2wTT0yxFttOAXN38_HdKzfbwCLcBGAs/s1600/218.jpg
```

**Pull request merged**